.env

```diff
@@ -1,3 +1,2 @@
-CUTTLE_ENV=dev
```

Comment:
@seriouslysean per our convo, nixing some of the environment-specific config while we sort out how we want this to work more precisely